plugins/commands/updateTrusted.js

```diff
@@ -35,14 +35,19 @@ module.exports = () => ({
             });
 
             if (commands.length === 0) {
-                throw boom.notFound(`Command ${namespace}/${name} does not exist`);
+                throw boom.notFound(
+                    `Command ${namespace}/${name} does not exist`
```

Comment:
idk if this rule makes sense to me

Reply:
there's a lot of extra newlines

Reply:
should make it consistent https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/41/files

Reply:
I will update this PR once that rule goes int